### DISCUSSION TITLE
Paypal Link for donations?

### DISCUSSION THREAD

**Discussion by Ep1cSpartan** [Maerciez/TheMillionaireGame#57] [Q&A] [answered]
Hi, 
I wanted to know if there is a paypal Link where I can send you something? you're putting so much work into this and I wanted to at least buy you a coffee or two xD

**Comment by Macronair** [comment, marked as answer]
At this point there is no PayPal link or other donation platform linked to this project.
Maybe in the (near) future I will consider it. But thanks again for the positive feedback!
